fortress/ros2_integration.md

```diff
@@ -35,6 +37,16 @@ The ROS message type is followed by an `@`, `[`, or `]` symbol where:
 Have a look at these [examples](https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_gz_bridge/README.md#example-1a-ignition-transport-talker-and-ros-2-listener)
 explaining how to make communication connections from ROS to Ignition and vice versa.
 
+It is also possible to use ROS Launch with the `ros_ign_bridge` and represent the topics in yaml format to be given to the bridge at launch time.
```

Comment:
I believe its also important to introduce how to use this in a more practical application early on as well